docs/data-sources/scalr_role.md

```diff
@@ -12,13 +12,23 @@ This data source is used to retrieve details of a single role by name and accoun
 
 ## Example Usage
 
+To retrieve an account manage role an account id and the role name is required, for example: 
```

Comment:
@emocharnik what's this? "account manage role"

Reply:
`account managed role` - it's a custom role the user create in the account